cmd/flightctl/main.go

```diff
@@ -34,6 +34,7 @@ func NewFlightCtlCommand() *cobra.Command {
 	cmd.AddCommand(cli.NewConsoleCmd())
```

Comment:
Additional note, 

If you add a:
   ./bin/flightctl login https://${IP}.nip.io:3443
in
    https://github.com/flightctl/flightctl/blob/main/test/scripts/deploy_with_helm.sh#L157

and then

remove the totality of:
https://github.com/flightctl/flightctl/blob/main/test/scripts/prepare_agent_config.sh
to use your ./bin/flightctl certificate request
command, leaving the agent config in the same place... you would have full e2e validation of your command in CI :)

Reply:
Nice! Thanks, I will try it

Reply:
I probably didn't implement this correctly because it caused CI to fail, but I can play around with it more separately to this PR.

Reply:
It is ok if you make it fail here I can help look at it.

Reply:
I took a closer look - in order to do this, I will need to place the agent config in a specific location in the `embedded` format, is this what we want? @fzdarsky was suggesting `stdout` for the output so that's what I implemented. We can certainly also have some option to put it in the file that CI expects, but as there have been so many changes to this PR it may be good to do it separately so that this can be merged. Thoughts?

Reply:
Ah, I misunderstood here because my brain is a bit fried. Totally possible and easy to do this so in `prepare_agent_config.sh` I've removed everything but the `cp` commands that we still need and I added the call to `./bin/flightctl/ certificate request > bin/agent/etc/flightctl/certs/config.yaml`. We'll see if it passes CI!

Reply:
https://github.com/flightctl/flightctl/actions/runs/11153840866/job/31002105193#step:7:1064

For the login failure you need to use the insecure tls flags (we have two, I never remember which one it is), otherwise we need to know the CA before hand, but for the devel env is completely fine to ignore that.